README.md

```diff
@@ -3,14 +3,17 @@
 This is a helper package that automatically installs and updates the [Svelte Language Server](https://github.com/sveltejs/language-tools) for you.
 
 
-
 ## Installation
 
 * Install [LSP](https://packagecontrol.io/packages/LSP), [Svelte](https://packagecontrol.io/packages/Svelte) and `LSP-svelte` from Package Control.
 * Restart Sublime.
 
 
-
 ## Configuration
 
 Open configuration file using command palette with `Preferences: LSP-svelte Settings` command or opening it from the Sublime menu (`Preferences > Package Settings > LSP > Servers > LSP-svelte`).
+
+
+## Enable for non-Svelte files
+
+Provides intellisense for Svelte files inside TS/JS files by installing the (typescript-svelte-plugin)[https://www.npmjs.com/package/typescript-svelte-plugin].
```

Comment:
I think that's not a correct statement. The server will not start for TS/JS files so it won't provide intellisense for those. Unless also the `selector` is changed. Should that be added here also?

Reply:
FWIW, I saw how other extensions implemented this:
both the VS code extension, and coc-svelte have this "contributes.typescriptServerPlugins" section in package.json: (still not sure how that  "contributes.typescriptServerPlugins" relates to the plugin being enabled in the server or not)

https://github.com/sveltejs/language-tools/blob/master/packages/svelte-vscode/package.json#L55
https://github.com/sveltejs/language-tools/blob/master/packages/svelte-vscode/src/tsplugin.ts#L25

```
commands.executeCommand('_typescript.configurePlugin', 'typescript-svelte-plugin', {
            enable
        });
```
> NOTE: theia ts ls has a 'configurePlugin' command, which I haven't tried, it maybe does the same thing as vs codes '_typescript.configurePlugin'

https://github.com/coc-extensions/coc-svelte/blob/master/package.json#L34
https://github.com/coc-extensions/coc-svelte/blob/master/src/tsplugin.ts#L34

----

I went for the laziest solution, 
and that it just to document how someone can have a good DX experience while working with svelte/ts/js files.

Reply:
> I think that's not a correct statement.

How about:
```
Install the (typescript-svelte-plugin)[https://www.npmjs.com/package/typescript-svelte-plugin].
Note that features that this plugin supports are only available within TS/JS files.
```

Reply:
Thanks for explaining. So then it looks like the typescript plugin is completely separate thing from this LSP and works on its own. I was confused when I read the `Enable for non-Svelte files` header as it's not really enabling this LSP but something completely separate. So my suggestion would be: 

```
## Svelte support in JS/TS files

The [typescript-svelte-plugin](https://www.npmjs.com/package/typescript-svelte-plugin) provides additional Svelte-related functionality in JS/TS files. To enable it, install it in your project and configure following its documentation.
```